src/utils/resource.ts

```diff
@@ -224,6 +224,14 @@ const deviceHardwareList: DeviceHardware[] = [
 		activelySupported: true,
 		displayName: "Raspberry Pi Pico",
 	},
+	{
+		hwModel: 47,
+		hwModelSlug: "RPI_PICO",
```

Comment:
Not sure where `hwModelSlug` is used for, but maybe it needs to be "RPI_PICOW"?

Reply:
It's technically supported to be the HW_MODEL protobuf property name, so RPI_PICO applies to both.